way_to_home/tests/unittests/place/testview.py

```diff
@@ -0,0 +1,268 @@
+        }
+        url = reverse('place', args=[])
+        response = self.client.post(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+
+    def test_post_empty_json(self):
+        """Method that tests unsuccessful post request with empty JSON data."""
+
+        data = {}
+        url = reverse('place', args=[])
+        response = self.client.post(url, json.dumps(data), content_type='application/json')
+
+        self.assertEqual(response.status_code, 400)
+
+    def test_put_success(self):
+        """Method that test success put request for the updating the certain task."""
+
+        data = {
+            'name': 'Work'
+        }
+
+        url = reverse('place', kwargs={'place_id': self.place.id})
+        response = self.client.put(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+    def test_put_non_owner(self):
+        """Method that tests for request to update non owner Place instance."""
+        another_user = CustomUser(id=102, email='another_user1@mail.com', is_active=True)
+        another_user.set_password('qwerty12345')
+        another_user.save()
+        self.client.login(email='another_user1@mail.com', password='qwerty12345')
+
+        data = {
+            'longitude': 12.842601
+        }
+        url = reverse('place', kwargs={'place_id': self.place.id})
+        response = self.client.put(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 403)
+
+    def test_put_empty_json(self):
+        """Method that tests unsuccessful put request with empty JSON data."""
+        data = {}
+        url = reverse('place', kwargs={'place_id': 11})
+        response = self.client.put(url, json.dumps(data), content_type='application/json')
+
+        self.assertEqual(response.status_code, 400)
+
+    def test_put_failed_wrong_id(self):
+        """Method that tests request to update non existent object."""
+
+        data = {
+            'stop_id': 15,
+        }
+
+        url = reverse('place', kwargs={'place_id': 1100})
+        response = self.client.put(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 404)
+
+    def test_put_non_id(self):
+        """Method that tests request to update object without id."""
+
+        data = {
+            'stop_id': 15,
+        }
+
+        url = reverse('place')
+        response = self.client.put(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+
+    def test_put_invalid_data(self):
+        """Method that tests unsuccessful put request with invalid data."""
+
+        data = {
+            'stop_id': "asd",
+        }
+        url = reverse('place', kwargs={'place_id': self.place.id})
+        response = self.client.post(url, json.dumps(data), content_type='application/json')
```

Comment:
```suggestion
        response = self.client.post(url, json.dumps(data), content_type='application/json')
```
I think it should be `put`

Reply:
Right!